README.md

```diff
@@ -1,65 +1,168 @@
+* As a logged in user
+* I want to interact with a live comment section for an event
+* So I can interact with comments in real-time
 
-- as a visitor
-- I want to understand what I can do with this app (or sign up, or log in)
-- so I can decide if I want to use it
 
 ### 3. Functionality
 
 The app's functionality includes:
 
-- Every User has the ability to create an account
-- Registered Users have the ability to log in
+* Every User has the ability to create an account
+* Registered Users have the ability to log in
+* Registered Users have the ability to create chat posts
+* Registered Users have the ability to view live chat windows
+* Registered Users have the ability to view chat posts from other users
 
 ### 4. Technology
 
-- Front-End: HTML5, CSS3, JavaScript ES6, React
+- Front-End: HTML5, CSS3, JavaScript, ES6, React
 - Back-End: Node.js, Express.js, Mocha, Chai, RESTful API Endpoints, Postgres
 - Development Environment: Heroku, DBeaver
 
 ### 5. Wireframes
 
 Landing Page
 :-------------------------:
-![Landing Page](./github-images/wireframes/landing-page-wireframe.png)
+![Landing Page](./github-images/wireframes/yamma-landing-wireframe.png)
 
 Register Page
 :-------------------------:
-![Register Page](./github-images/wireframes/register-page-wireframe.png)
+![Register Page](./github-images/wireframes/yamma-sign-up-wireframe.png)
 
-Login Page
+ChatPage
 :-------------------------:
-![Login Page](./github-images/wireframes/login-page-wireframe.png)
+![Chat Page](./github-images/wireframes/yamma-chat-2-wireframe.png)
+![Chat Page](./github-images/wireframes/yamma-chat-wireframe.png)
+
+ArticlePage
+:-------------------------:
+![Article Page (by region)](./github-images/wireframes/yamma-article-by-region-wireframe.png)
 
 ### 6. Front-end Structure - React Components Map
+* __Index.js__ 
+* __App.js__ (stateful)
+* __FilterRoute__ (context-provider)
+* __DashboardRoute.js__ (stateful)
+* __Map.js__ (stateless)
+* __EventList.js__ (stateful) (no context)- connect to events table
+* __EventCard.js__ (stateless)
+* __Sidebar.js__ (stateful)
+* __SearchTab.js__ (stateful)
+* __IconList.js__ (stateless)
+* __Icon.js__ (stateless)
+* __CategoryList.js__ (stateless) - scrollable
+* __EventPageRoute.js__ (stateful) 
+* __ArticlePanel.js__(stateful)
+* __ChatPanel.js__ (stateful) - connect to comments table [event_id]
+* __ChatFeed.js__ (stateful)
+* __ChatForm.js__ (stateful) - connect to comments table
+* __RelatedArticleList.js__(stateful)
+* __RelatedArticleCard.js__(stateful)
+* __LoginRoute.js__ (stateful)
+* __LoginForm.js__ (stateful) - connect to users table [username, password]
+* __RegisterRoute.js__(stateful) 
+* __RegisterForm.js__ (stateful) - connect to users table [username, email, password]
+* __Header.js__ (stateless)
+* __Navbar.js__(stateful)
+* __Footer.js__ (stateless)
 
-### 7. Back-end Structure - Business Objects
 
+### 7. Back-end Structure - Business Objects
```

Comment:
make this a link to the back end repo:
### [Back-end](https://github.com/MettyS/yamma-server)